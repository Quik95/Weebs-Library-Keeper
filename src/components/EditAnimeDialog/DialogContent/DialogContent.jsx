import React from 'react';
import PropTypes from 'prop-types';

// MUI
import MuiDialogContent from '@material-ui/core/DialogContent';
import CardMedia from '@material-ui/core/CardMedia';
import Grid from '@material-ui/core/Grid';

// styles
import useStyles from './DialogContent.style';

// Custom components
import EditAnimeForm from './EditAnimeForm/EditAnimeForm';

function DialogContent({ animeId }) {
  const classes = useStyles();

  return (
    <MuiDialogContent dividers>
      <Grid container spacing={4}>
        <Grid item xs={12}>
          <CardMedia
            image="https://s4.anilist.co/file/anilistcdn/media/anime/cover/large/bx105333-5p1MKBlGxZFF.jpg"
            className={classes.animeThumbnail}
          />
        </Grid>
        <Grid item xs={12}>
          <EditAnimeForm />
        </Grid>
      </Grid>
    </MuiDialogContent>
  );
}

DialogContent.propTypes = {
  animeId: PropTypes.string.isRequired,
};

export default DialogContent;
