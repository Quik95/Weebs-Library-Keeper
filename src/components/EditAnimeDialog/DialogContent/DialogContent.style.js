import { makeStyles } from '@material-ui/core/styles';

export default makeStyles(theme => ({
  animeThumbnail: {
    height: 250,
  },
}));
