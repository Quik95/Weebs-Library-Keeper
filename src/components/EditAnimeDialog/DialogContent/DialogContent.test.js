import DialogContent from './DialogContent';

describe('<DialogContent/>', () => {
  const wrapper = shallow(<DialogContent animeId="198cndud198nu892281" />);

  it('Renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
