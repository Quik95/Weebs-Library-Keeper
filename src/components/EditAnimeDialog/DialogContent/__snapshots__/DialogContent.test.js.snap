// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DialogContent/> Renders correctly 1`] = `<div />`;
